code/datums/interior/map_templates.dm

```diff
@@ -0,0 +1,12 @@
+/datum/map_template/interior
+	name = "Base Interior Template"
+	var/prefix = "_maps/interiors/"
```

Comment:
autodoc, also is it really useful to split this into a prefix and filename var instead of just 1 filename var lol? The whole combining it thing seems kinda pointless

Reply:
Shuttles does this, it can be ok to decrease the amount of copy paste at least